internal/app/exporter/exporter_json_test.go

```diff
@@ -141,7 +141,6 @@ func TestHandlesOverWrittingLongString(t *testing.T) {
 func TestWriteRow(t *testing.T) {
 	tmpExporter := getTemporaryJSONExporter()
 	str := `{"abc": 123}`
-	var tmp json.RawMessage
-	tmp = []byte(str)
+	var tmp json.RawMessage = []byte(str)
```

Comment:
```suggestion
	tmp := []byte(str)
```

Reply:
`tmpExporter.WriteRow` expects `json.RawMessage` type as its parameter. 
 `json.RawMessage` is `[]byte` underneath but we still have to declare the type.